core/Pos/Binary/Core/Update.hs

```diff
@@ -139,8 +134,12 @@ instance HasConfiguration => Bi U.UpdateVote where
         uvProposalId <- decode
         uvDecision   <- decode
         uvSignature  <- decode
-        toCborError $ over _Left ("decode@UpdateVote: " <>) $
-            U.validateUpdateVote U.UnsafeUpdateVote{..}
+        pure U.UnsafeUpdateVote{..}
+        {-
+        case U.validateUpdateVote U.UnsafeUpdateVote{..} of
+            Left err -> fail $ toString ("decode@UpdateVote: " <> err)
+            Right uv -> pure uv
+        -}
```

Comment:
This code is commented out and should be deleted, right?

Reply:
Done